htlcswitch/link.go

```diff
@@ -1932,6 +1964,7 @@ func (l *channelLink) handleUpstreamMsg(msg lnwire.Message) {
 		err = l.channel.ReceiveNewCommitment(&lnwallet.CommitSigs{
 			CommitSig:  msg.CommitSig,
 			HtlcSigs:   msg.HtlcSigs,
+			PartialSig: msg.PartialSig,
```

Comment:
can panic in `VerifyCommitSig` if `msg.PartialSig` is `nil` for a taproot channel

Reply:
`VerifyCommitSig` now checks if the sig is nil or not.

Reply:
`VerifyCommitSig` now checks if the sig is nil or not.